engineapi/engine.go

```diff
@@ -148,8 +159,12 @@ func (e *EngineBinary) ReplicaAdd(engine *longhorn.Engine, replicaName, url stri
 		}
 	}
 
+	if version.ClientVersion.CLIAPIVersion >= 9 {
```

Comment:
NIT: It's better to use a const with a comment explaining the reason instead. e.g.,:
```
// CLIAPIVersionNine is the first version of instance identity validation, which is implemented by the gRPC identity interceptor.
CLIAPIVersionNine = 9
```

Reply:
This makes sense to me in general, but it would be a break from the style/convention established previously in the code block and others.

Reply:
I am fine with either style.